rust/MEMORY_PRESSURE_FIXES.md

```
@@ -0,0 +1,205 @@
+your_ai train --model llama-8b --max-memory 70.0
+```
+Look for output:
+```
+⚠️  No memory limit specified. Auto-detecting safe limit: 70.0 GB
+   (Based on 96.0 GB available system memory)
+🔒 Set MLX memory limit to 70.0 GB (was 6.3 GB)
+```
+
+### Verify Training Speed:
+Look for progress bar output like:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)